src/oc-id/app/views/sessions/new.html.erb

```diff
@@ -1,31 +1,18 @@
-<div>
+<div class="medium-6 columns">
   <h1>Sign In</h1>
-</div>
-<div>
-  <%= form_tag(auth_chef_callback_path, :method => :post, 'data-abide' => '') do %>
-    <div class="row">
-      <div class="medium-6 columns username-field">
-        <%= label_tag :username %>
-        <%= text_field_tag :username, nil, :autofocus => true, :required => true %>
-        <small class="error">Required</small>
-      </div>
-    </div>
-    <div class="row">
-      <div class="medium-6 columns password-field">
-        <%= label_tag :password %>
-        <%= password_field_tag :password, nil, :required => true %>
-        <small class="error">Required</small>
-      </div>
-    </div>
-    <div class="row">
-      <div class="medium-6 columns">
+  <p>Please sign in with your hosted Chef account.</p>
```

Comment:
This isn't going to be a "hosted Chef" account on-prem. In Manage we have a `chef_name` configuration option. I wonder if there should be one here (or at the top level of Chef server.)

I suppose if you wanted to defer that decision you could just say "Sign into your Chef account" here.